test/katex-spec.js

```diff
@@ -88,115 +87,100 @@ const parseAndSetResult = function(expr, result, settings) {
-                },
+        toNotParse: function(actual, settings) {
+            const usedSettings = settings ? settings : defaultSettings;
+
+            const result = {
+                pass: false,
+                message: "Expected '" + actual + "' to fail " +
+                    "parsing, but it succeeded",
             };
+
+            try {
+                parseTree(actual, usedSettings);
+            } catch (e) {
+                if (e instanceof ParseError) {
+                    result.pass = true;
+                    result.message = "'" + actual + "' correctly " +
+                        "didn't parse with error: " + e.message;
+                } else {
+                    result.message = "'" + actual + "' failed " +
+                        "parsing with unknown error: " + e.message;
+                }
+            }
+
+            return result;
         },
 
-        toBuild: function() {
-            return {
-                compare: function(actual, settings) {
-                    const usedSettings = settings ? settings : defaultSettings;
-
-                    const result = {
-                        pass: true,
-                        message: "'" + actual + "' succeeded in building",
-                    };
-
-                    expect(actual).toParse(usedSettings);
-
-                    try {
-                        _getBuilt(actual, settings);
-                    } catch (e) {
-                        result.pass = false;
-                        if (e instanceof ParseError) {
-                            result.message = "'" + actual + "' failed to " +
-                                "build with error: " + e.message;
-                        } else {
-                            result.message = "'" + actual + "' failed " +
-                                "building with unknown error: " + e.message;
-                        }
-                    }
-
-                    return result;
-                },
+        toBuild: function(actual, settings) {
+            const usedSettings = settings ? settings : defaultSettings;
+
+            const result = {
+                pass: true,
+                message: "'" + actual + "' succeeded in building",
             };
+
+            expect(actual).toParse(usedSettings);
+
+            try {
+                _getBuilt(actual, settings);
+            } catch (e) {
+                result.pass = false;
+                if (e instanceof ParseError) {
+                    result.message = "'" + actual + "' failed to " +
+                        "build with error: " + e.message;
+                } else {
+                    result.message = "'" + actual + "' failed " +
+                        "building with unknown error: " + e.message;
+                }
+            }
+
+            return result;
         },
 
-        toParseLike: function(util, baton) {
-            return {
-                compare: function(actual, expected, settings) {
```

Comment:
I don't know how `settings` was being passed into this function.  I should probably remove it to avoid confusion.